src/config.js

```diff
@@ -67,7 +67,12 @@ function getScannerParams(basePath, params = {}) {
  *  - httpOptions, if proxy
  */
 function getExecutableParams(params = {}) {
-  const config = {};
+  const config = {
+    httpOptions: {
```

Comment:
I don't see the point of declaring this default empty structure, since we can easily use `?.` later to access if when needed.